src/citation/views/citation_project_view.py

```diff
@@ -1,13 +1,51 @@
+from rest_framework import status
 from rest_framework.decorators import action
-from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
 from rest_framework.viewsets import ModelViewSet
+from django.db.models import Q
 
 from citation.models import CitationProject
 from citation.serializers import CitationProjectSerializer
+from user.related_models.organization_model import Organization
 
 
 class CitationProjectViewSet(ModelViewSet):
     queryset = CitationProject.objects.all()
     permission_classes = [IsAuthenticated]
     serializer_class = CitationProjectSerializer
     ordering = ["-updated_date"]
+
+    def list(self, request):
```

Comment:
We should make this as an `@action` endpoint, so we can take in an pk for organization